.gitignore

```diff
@@ -11,6 +11,10 @@
 !/bin
 /bin/*
 !/bin/brew
+!/share
+/share/*
+!/share/doc
+/share/doc/*
```

Comment:
I'd love if our `.gitignore` could be less weird. If anyone feels like doing so ever: feel free.